File: Modul-6/Soal-3/PRAK602-2410817110016-MuhammadAzriRahman.py
print("Input")
n1, n2 = map(int, input("").split())

if n1 != n2:
    print("Jumlah tidak sama")
else:
    row1 = list(map(int, input().split()))

    row2 = list(map(int, input().split()))

    print("Output")
    for i in range(n1):
        print(row1[i]*row2[i], end=" ")

File: Modul-6/Soal-3/PRAK602-2410817110016-MuhammadAzriRahman.c
#include <stdio.h>

void main() {
    int n1, n2;
    printf("Input\n");
    scanf("%d %d", &n1, &n2);

    if (n1 != n2) {
        printf("Jumlah tidak sama\n");
    } else {
        int array[2][n1];

        for (int i = 0; i < 2; i++) {
            for (int j = 0; j < n1; j++) {
                scanf("%d", &array[i][j]);
            }
        }
        printf("Output\n");
        for (int j = 0; j < n1; j++) {
            printf("%d ", array[0][j] * array[1][j]);
        }
        printf("\n");
    }

}
